R/map.R

```diff
@@ -188,44 +187,51 @@ map.ppi <- function(x, map, param, alpha = 0.7, xlim, ylim, zlim = c(-20, 20),
   }
 
   # convert data values to hex color string values.
-  r@data@values <- col_func(r@data@values, zlim)
+  #r@data@values <- col_func(r@data@values, zlim)
+
+  rdf <- raster::as.data.frame(r, xy = TRUE)
+  names(rdf) <- c("x", "y", "fill")
+  rdf$fill <- col_func(r@data@values, zlim)
 
   # these declarations prevent generation of NOTE "no visible binding for
   # global variable" during package Check
-  lon <- lat <- y <- z <- NA
+  fill <- lon <- lat <- X <- Y <- y <- z <- NA
   # extract unique radar locations
-  latlon_radar <- unique(data.frame(lat=c(x$geo$lat), lon=c(x$geo$lon)))
```

Comment:
lat and lon attributes can be vectors when using `composite_ppi()`, need to double check compatability with that function. When compositing multiple radars in a ppi, function needs to plot radar locations of all radars